app/uk/gov/hmrc/economiccrimelevyregistration/controllers/AmendRegistrationStartController.scala

```diff
@@ -40,7 +42,8 @@ class AmendRegistrationStartController @Inject() (
   errorHandler: ErrorHandler,
   view: AmendRegistrationStartView,
   registrationService: EclRegistrationService,
-  registrationConnector: EclRegistrationConnector
+  registrationConnector: EclRegistrationConnector,
```

Comment:
why are we using the connector here rather than registrationService?

Reply:
Done